.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        description: "Version to use for the release."
+        required: true
+        default: "X.Y"
+      releaseBody:
+        description: "Information about the release"
+        required: true
+        default: "New release"
+jobs:
+  Release:
+    environment: Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: main
+      - name: Set up python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+      - name: Build
+        run: make compile-with-docker-all
+      - name: Run Integ Tests
+        run: |
+          make tests-with-docker
+          make integ-tests-with-docker-x86-64
+          make integ-tests-with-docker-arm64
+          make integ-tests-with-docker-old
```

Comment:
Actually, these will compile each binary again (after you already built them before).

It should be enough to run
```
make tests-with-docker
make integ-tests
```